src/main/docs/guide/config/configurationProperties.adoc

```diff
@@ -36,6 +36,41 @@ For the cases where the configuration properties class inherits properties from
 
 The names supplied to the includes/excludes list must be the "property" name. For example if a setter method is injected, the property name is the de-capitalized setter name (`setConnectionTimeout` -> `connectionTimeout`).
 
+[[configurationPropertiesAccessorsStyle]]
+== Change accessors style
+
+Since 3.3, Micronaut supports defining different accessors prefixes for getters and setter other than the default `get` and `set` defined for Java Beans. Annotate your POJO or `@ConfigurationProperties` class with the link:{api}/io/micronaut/core/annotation/AccessorsStyle.html[@AccessorsStyle] annotation.
+
+This is useful when you write the getters and setters in a fluent way or when using libraries like Lombok that can generate those fluent accessors. For example:
+
+.Using `@AccessorsStyle`
+[source,java]
+----
+import io.micronaut.context.annotation.ConfigurationProperties;
+import io.micronaut.core.annotation.AccessorsStyle;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
```

Comment:
We shouldn't encourage the use of Lombok in our docs, remove references to Lombok and just use a normal class